kuma/static/styles/components/contribution/banner.scss

```diff
@@ -0,0 +1,99 @@
+@import 'includes/vars';
```

Comment:
I added this ``@import`` to fix a linting issue, I'm not sure if it is required. It's not upstream - https://github.com/mozilla/kuma/blob/potato-1490727/kuma/static/styles/components/contribution/banner.scss

Reply:
We understood this to denote this file's dependencies. But not necessary, can be removed.